julearn/pipeline.py

```diff
@@ -185,11 +184,8 @@ def transform_target(self, X, y):
         return y_true
 
     def preprocess(self, X, y):
-        if self.y_transformer is not None:
-            y_trans = self.y_transformer.fit_transform(X, y)
-        else:
-            y_trans = y
-
+        X_conf_trans = self.transform_confounds(X)
+        y_trans = self.transform_target(X_conf_trans, y)
```

Comment:
Is it ok like this? I find it weird that in order to transform the target we need to give `X_conf_trans`.

Or either `transform_target` and `transform_confouds` are private and the user should not call them or `transform_target` does all the required transforms to properly transform the target.

Otherwise, the user can transform the target without doing confounds.

Reply:
We will adjust it as mentioned in #17 . 
I will merge this now and create a new PR, because this is another issue.